torch/testing/_internal/common_methods_invocations.py

```diff
@@ -19112,6 +19118,36 @@ def __init__(
             DecorateInfo(unittest.expectedFailure, 'TestBinaryUfuncs', 'test_type_promotion'),
         )
     ),
+    ElementwiseBinaryPythonRefInfo(
+        "_refs.fmax",
+        torch_opinfo_name="fmax",
+        supports_rhs_python_scalar=False,
+        skips=(
+            # refs.fmax supports scalars, unlike torch.fmax
```

Comment:
Yes! ... but ... today that support is spotty. I'm planning to address that this week by updating all the prims to support scalars and scalar tensors, and then refs should inherit the support, too. 

So for now I wouldn't worry too much about working with scalars or not because a fix is coming